slurm_ops_manager/templates/mellanox_repo_centos7.repo

```diff
@@ -0,0 +1,15 @@
+#
+# Mellanox Technologies Ltd. public repository configuration file.
+# For more information, refer to http://linux.mellanox.com
+#
+# Downloaded from
+# https://linux.mellanox.com/public/repo/mlnx_ofed/latest/rhel7.9/mellanox_mlnx_ofed.repo
```

Comment:
`rhel7.9` - the `7.9` representing the os version should be a template variable filled in by the charm when it writes out this file